src/presentational-components/shared/content-list.js

```diff
@@ -0,0 +1,77 @@
+import React, { Fragment, useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Text, TextVariants } from '@patternfly/react-core';
+import { Table, TableHeader, TableBody, TableVariant } from '@patternfly/react-table';
+
+import { ListLoader } from '../../presentational-components/shared/loader-placeholders';
+
+const NoItems = () => (
+  <Text component={ TextVariants.h1 }>No items found</Text>
+);
+
+const ContentList = ({ data, columns, isCompact, isLoading, actionResolver, areActionsDisabled,
+  borders, routes,  titlePlural, renderEmptyState }) => {
+
+  const [ rows, setRows ] = useState([]);
+
+  useEffect(() => {
+    setRows(data);
+  }, [ data ]);
+
+  const renderEmptyContent = () => renderEmptyState ? renderEmptyState()
+    : <NoItems/>;
+
+  const renderContent = () =>
+    (!rows || rows.length === 0) ?
```

Comment:
`!rows` should never happen. Default is empty array and `data` should have default prop of `[]`